src/arch/xtensa/include/arch/sof.h

```diff
@@ -59,8 +59,11 @@ static inline void fill_core_dump(struct sof_ipc_dsp_oops_xtensa *oops,
 				  uint32_t ps, uintptr_t stack_ptr,
 				  uintptr_t *epc1)
 {
+	uint32_t oops_size = sizeof(struct sof_ipc_dsp_oops_xtensa)
```

Comment:
Seems to be a good candidate for macro since the formula is repeated in a couple of places.